docs/how-to/create-a-router/configure-ensembler.md

```diff
@@ -26,7 +26,14 @@ It is not possible to select as the fallback response a route that has traffic r
 {% endhint %}
 
 ## Docker
-Turing will deploy the specified image as a post-processor and will send in the request payload the following, for ensembling - the original request, responses from all routes, and the treatment configuration (if an Experiment Engine is selected, in Configure Experiment Engine). The ensembler's request headers will contain the original request headers sent to Turing, merged with the enricher's response headers (if there are duplicates, the value in the enricher's response headers will take precedence). To configure a Docker ensembler, there are 3 sections to be filled.
+
+Turing will deploy the specified image as a post-processor and will send in the request payload the following, for 
+ensembling - the original request, responses from all routes, and the treatment configuration (if an Experiment 
+Engine is selected, in the Configure Experiment Engine step). The ensembler's request headers will contain the original 
+request headers sent to Turing, merged with the enricher's response headers (if there are duplicates, the value in 
+the enricher's response headers will take precedence), and an identifier `turing_req_id` that is uniquely assigned to each request received by the Router. 
```

Comment:
Same comments as with `docs/how-to/create-a-router/configure-enricher.md` - missing "is" and `Router` -> `Ensembler`; `turing_req_id` -> `Turing-Req-ID`.